src/v/archival/probe.h

```diff
@@ -52,8 +65,14 @@ class ntp_level_probe {
     int64_t _missing = 0;
     /// Width of the offset range yet to be uploaded
     int64_t _pending = 0;
+    /// Number of segments deleted by garbage collection
+    int64_t _segments_deleted = 0;
+    /// Current start offset of the manifest
+    model::offset _start_offset{0};
```

Comment:
This seems to be unused (+ we probably don't want it in public_metrics due to cardinality and the fact that it's a raft offset that doesn't make sense to the user)

Reply:
Correct. I missed removing it when I re-did the metrics bit.

Reply:
Done.